api/CSSValue.json

```diff
@@ -31,10 +31,10 @@
             "version_added": null
           },
           "safari": {
-            "version_added": null
+            "version_added": "≤4"
```

Comment:
Confirmed with http://mdn-bcd-collector.appspot.com/tests/api/CSSValue that it's still there in STP. A Safari-only API at this point then, interesting!